ui_framework/src/components/dial_menu/_index.scss

```diff
@@ -0,0 +1,5 @@
+// Dial menu is a phone like dial comprised of an svg icon and a title.
+
+$kuiDialMenuWidth: $kuiSize * 6;
```

Comment:
When I see the word "dial", I see [these kinds of dials](https://www.google.com/search?q=dial&safe=active&rlz=1C5CHFA_enUS695US695&source=lnms&tbm=isch&sa=X&ved=0ahUKEwi3toXSipvVAhWBeSYKHfWoD98Q_AUICigB&biw=2560&bih=1315). Of all those images I think I only saw two which are the kind of dial we're trying to reference here, so I'm worried this won't be as clear a name as we'd like.

How about `GridMenu`, `HashMenu`, `BanchanMenu`, `ChocolateBoxMenu`, or `TiledMenu` instead?